### PR TITLE
Install and use standard

### PR DESCRIPTION
**Pull request by kizza**
No more red marks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run react-project:start:prod || npm run react-project:start:dev",
     "test": "karma start",
-    "react-project:start:dev": "eslint modules && react-project start",
+    "lint": "node_modules/.bin/standard --verbose | snazzy",
```
Comment (bradparker) [inline package.json]: `npm run` is magic. You don't need `node_modules/.bin/x`

Comment (kizza) [inline package.json]: Huh - for some reason we did in postoffice I thought - will remove!

File: modules/routes.js [hunk 2]
```diff
@@ -9,12 +9,12 @@ import SimpleForm from './components/SimpleForm'
...
 export default (
   <Route>
-    <Route path="/" component={App}>
-      <IndexRoute component={SimpleForm}/>
+    <Route path='/' component={App}>
+      <IndexRoute component={SimpleForm} />
     </Route>
-    <ServerRoute path="/api">
-      <ServerRoute path=":hello" get={hello}/>
+    <ServerRoute path='/api'>
```
Comment (bradparker) [inline modules/routes.js]: Can probably remove these if you like? While it's getting a look in.

File: package.json (hunk 1 above)
Comment (kizza) [inline package.json]: Actually of course - it was bash scripts that got me in that habit!

File: modules/routes.js (hunk 2 above)
Comment (kizza) [inline modules/routes.js]: True - can remove all the api stuff (though it's pretty cool react-project wise :)

**Pull request merged**